jcache/embedded/src/main/java/org/infinispan/jcache/embedded/JCache.java

```diff
@@ -220,8 +228,8 @@ public V get(final K key) {
       // ReadWriteMap.evalMany is not that useful since it forces us to transfer keys
       List<? extends SimpleEntry<? extends K, CompletableFuture<R>>> list =
             keys.stream()
-                .map(k -> new SimpleEntry<>(k, map.eval(k, function)))
-                .collect(Collectors.toList());
+                  .map(k -> new SimpleEntry<>(k, map.eval(k, function)))
+                  .collect(Collectors.toList());
```

Comment:
Please reformat only changed lines!

I've seen quite a few whitespace changes on unmodified lines like this, but this jumped to me because I have `Chained method calls / Align when multiline` enabled in IntelliJ, and if you have it disabled, formatting the entire file changes the indent of all my lines.

Reply:
Maybe you could check the Intellij style and update it? https://issues.redhat.com/browse/ISPN-11588

Reply:
One day :)